app/task_routes.py

```diff
@@ -0,0 +1,140 @@
+from flask import Blueprint, abort, jsonify, make_response, request
+from app import db
+from app.models.task import Task
+from datetime import datetime
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body:
+        return {"details": "Invalid data"}, 400
+
+    new_task = Task.from_dict(request_body)
+    db.session.add(new_task)
+    db.session.commit()
+
+    return {
+        "task": new_task.to_dict()
+    }, 201
+
+
+@task_bp.route("", methods=["GET"])
+def read_all_tasks():
+    tasks = Task.query.all()
+    sort_request = request.args.get("sort") #Added this
+    task_list = []
+    
+    """HELPER FUNCTION TO DETERMINE IF TASK IS COMPLETED"""
+    def is_complete():
+        if "completed_at" in task_list == None:
+            return True
+        else:
+            return False
```

Comment:
For this project we don't need to create a method within route method. Your to_dict() method in Task already handles computing the is_complete method. So lines 43-47 here repeat lines 14-17 in task.py

Please take a moment to review the to_dict() method in Task